src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/DSAKeyValue.cs

```diff
@@ -23,7 +23,7 @@ public DSAKeyValue()
             _key = DSA.Create();
         }
 
-        public DSAKeyValue(DSA key)
+        public DSAKeyValue(DSA? key)
```

Comment:
this might be worth filing an issue (or adding to existing one) since we accept null here but later throw if it's null. Unless any of the code assumes this can be null I'd go ahead and change annotation to non-null to avoid breaking change in the future

Reply:
Good point. Line 81 would throw an NRE, so I've changed the annotation to be non null as suggested.